iOS Client/Models/MBPodcast.swift

```diff
@@ -29,4 +29,5 @@ struct Podcast: Detailable {
     let summary: String?
     let pubDate: Date
     let title: String?
+    let podTitle: String
```

Comment:
`feedName` `streamName` or `channelName`? podTitle seems a little ambiguous

Reply:
strong -- will change this